File: app/src/main/java/com/drivingschoolrwandaapp/models/response/SubscriptionPlansResponse.java
package com.drivingschoolrwandaapp.models.response;

import com.drivingschoolrwandaapp.models.entities.SubscriptionPlan;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SubscriptionPlansResponse {

    @SerializedName("success")
    private boolean success;

    @SerializedName("data")
    private List<SubscriptionPlan> data;

    public boolean isSuccess() {
        return success;
    }

    public List<SubscriptionPlan> getData() {
        return data;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/response/SubscriptionPlansResponse.kt
package com.drivingschoolrwandaapp.models.response

import com.drivingschoolrwandaapp.models.entities.SubscriptionPlan
import com.google.gson.annotations.SerializedName

data class SubscriptionPlansResponse(
    @SerializedName("success") var success: Boolean = false,
    @SerializedName("data") var data: List<SubscriptionPlan>? = null
)
